domains/thenamelessdev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "thenamelessdev",
+    "discord": "thenamelessdev"
+  },
+  "records": {
+    "CNAME": "thenamelessdev.github.io/about-me"
```

CNAME doesnt allow url trailing (/)
```suggestion
    "CNAME": "thenamelessdev.github.io"
```